modules/tools/sensor_calibration/extract_static_data.py

```diff
@@ -46,6 +46,11 @@
 def mkdir_p(path):
     if not os.path.isdir(path):
         os.makedirs(path)
+def get_subfolder_list(d):
+    """list the 1st-level directories under the root directory
+    ignore hidden folders"""
+    return [f for f in os.listdir(d) if not f.startswith('.') and\
```

Comment:
`\` is not needed when it's inside a `[]`, `()`, `{}`, etc. Please also fix others.